apps/stable/cluster-autoscaler/values.yaml.gotmpl

```diff
@@ -0,0 +1,28 @@
+{{- if eq "eks" .Values.jxRequirements.cluster.provider }}
+autoDiscovery: 
+  clusterName: {{ .Values.jxRequirements.cluster.provider }}
+  tags:
+  - k8s.io/cluster-autoscaler/enabled
+  - k8s.io/cluster-autoscaler/{{ .Values.autoDiscovery.clusterName }}
+awsRegion: {{ .Values.jxRequirements.cluster.region }}
+awsAccessKeyID: ""AKIAYBCHSJM224CUEI6T"
+awsSecretAccessKey: "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"
+image:
+  repository: k8s.gcr.io/cluster-autoscaler
+  tag: v1.17.1
+  pullPolicy: IfNotPresent
+rbac:
+  create: true
+  serviceAccount:
+    create: true
+    name: cluster-autoscaler
+    serviceAccountAnnotations: eks.amazonaws.com/role-arn: arn:aws:iam::{{ .Values.jxRequirements.cluster.project }}:role/{{ .Values.jxRequirements.cluster.clusterName }}-cluster-autoscaler-cluster-autoscaler
+
```

Comment:
and remove all of these additional line breaks. Just one line break after the last non empty line will do. Also make sure there are no trailing whitespaces and ensure indentation is right. That's very crucial to doing yaml right